Tests/TestingTests/Traits/CustomTraitTests.swift

```diff
@@ -8,11 +8,75 @@
 // See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 //
 
-// Note: Do NOT include `@_spi` on this import. The contents of this file are
-// specifically intended to validate that a type conforming to `TestTrait` can
-// be declared without importing anything except the base.
-import Testing
+@testable @_spi(ExperimentalTestRunning) @_spi(ExperimentalEventHandling) import Testing
 
 // This is a "build-only" test which simply validates that we can successfully
 // declare a type conforming to `TestTrait` and use its default implementations.
 struct MyCustomTrait: TestTrait {}
+
+private struct CustomCountTrait: CustomExecutionTrait, TestTrait {
```

Comment:
Since this trait no longer _counts_ anything, let's rename it to something more appropriate. Since it's in a test, it doesn't need to be perfect API, but it should at least have a name that's internally consistent. :)